openlane/steps/step.py

```diff
@@ -939,6 +994,13 @@ def run_subprocess(
                     current_rpt.write(line)
                 elif not silent and "table template" not in line:  # sky130 ff hack
                     verbose(line.strip(), markup=False)
+        process_stats_thread.join()
+        with open(os.path.join(self.step_dir, "process_stats.json"), "w") as f:
```

Comment:
If multiple processes run in the same step, `process_stats.json` will be overriden

Reply:
May be a good idea to use the `log_to` value and replace the extension with `.stats.json` or something